packages/wallets/src/components/WalletCardIcon/WalletCardIcon.scss

```diff
@@ -0,0 +1,25 @@
+.wallets-card-icon {
+    &-xs {
+        &--rounded {
+            transform: scale(50%);
+        }
+    }
+
+    &-sm {
+        &--rounded {
+            transform: scale(60%);
+        }
+    }
+
+    &-md {
+        transform: scale(100%);
```

Comment:
Do we have a size handled for mobile here, or is this scaling `100%` fine for mobile?

Reply:
the width for mobile and desktop handled iin component already, so scaling is the same for both